docs/reference/contributing/connectivity/MeshInterface.md

```diff
@@ -170,6 +170,67 @@ State|Description
 
 ![tx](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/tx_process.png)
 
+### MESH PHY extended driver support
+
+Mesh IEEE 802.15.4 MAC will automatically use RF driver extended features. These features are needed when MAC user need to send IEEE 802.15.4 frames. MAC extensuon will enable if driver not support extended features. 
```

Comment:
Mesh IEEE 802.15.4 MAC will automatically use RF driver extended features if those are enabled in driver.

These features are needed when MAC user need to send IEEE 802.15.4 frames. -> this is not right?

MAC extensuon will enable if driver not support extended features.  -> MAC extensions are used automatically if driver implements the extension. If these extensions are not implemented the default behaviour is used thus limiting features.

Reply:
ok